pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -731,4 +730,8 @@ public static class Range {
       public static final String UPPER_UNBOUNDED = DELIMITER + UNBOUNDED + UPPER_EXCLUSIVE;
     }
   }
+
+  public static class IdealState {
+    public static final String QUERY_TIME_BOUNDARY = "query.time.boundary";
```

Comment:
(optional) I feel `TIME_BOUNDARY` is more concise. To be more specific, `HYBRID_TABLE_TIME_BOUNDARY` might also be good

Reply:
Ack